apps/vm_species.c

```diff
@@ -100,21 +109,77 @@ double
 vm_species_rhs(gkyl_vlasov_app *app, struct vm_species *species,
```

Comment:
We should refactor this function to move the LBO terms to their own file. This way we will avoid these nested conditionals.